ui/src/app/shared/components/application-selector/application-selector.tsx

```diff
@@ -18,7 +20,9 @@ export const ApplicationSelector = ({apps, formApi}: {apps: models.Application[]
                     <label key={appInstanceName(app)} style={{marginTop: '0.5em', cursor: 'pointer'}}>
                         <CheckboxField field={`app/${i}`} />
                         &nbsp;
-                        {app.isAppOfAppsPattern ? `(App of Apps) ${appQualifiedName(app)}` : appQualifiedName(app)}
+                        {app.isAppOfAppsPattern
```

Comment:
side note, I never seen the `(App of Apps)` prefix so far

Reply:
I've not seen the `App of Apps`  ui myself.